tests/fuzzers/bls12381/bls12381_test.go

```diff
@@ -51,9 +51,9 @@ func FuzzG1Add(f *testing.F) {
 	})
 }
 
-func FuzzG1Mul(f *testing.F) {
+func FuzzCrossG1Mul(f *testing.F) {
```

Comment:
If you change these names, or add new targets, you need to also update `oss-fuzz.sh` accordingly

Reply:
I think you should just leave this name be instead
```suggestion
func FuzzG1Mul(f *testing.F) {
```

Reply:
Ping on this

Reply:
> I think you should just leave this name be instead

The PR expands these to be cross-library fuzzers.  Doesn't it make sense to have consistent naming with the other fuzzers of the same type?

Reply:
Ok, but if so you need to update `oss-fuzz.sh`